Makefile

```diff
@@ -236,7 +236,7 @@ start-stress-test: zetanode
 #TODO: replace OLD_VERSION with v16 tag once its available
 zetanode-upgrade: zetanode
 	@echo "Building zetanode-upgrade"
-	$(DOCKER) build -t zetanode:old -f Dockerfile-localnet --target old-runtime --build-arg OLD_VERSION='release/v16' .
+	$(DOCKER) build -t zetanode:old -f Dockerfile-localnet --target old-runtime --build-arg OLD_VERSION='release/v17' .
```

Comment:
The upgrade test should trigger the migration now when we create a release. 

I'm unsure of the best way to test an upgrade when merging migration scripts other than unit tests. Any ideas are welcome

Reply:
> other than unit tests

You mean E2E tests?

Reply:
With E2E tests, I think we might need to consider a Make command similar to
```
start-upgrade-test-light
```
but running admin tests instead 
```
start-upgrade-test-admin
```